templates/typescript_gapic/_util.njk

```diff
@@ -263,28 +263,41 @@ request.{{ oneComment.paramName.toCamelCase() }}
 {%- endmacro -%}
 
 {%- macro initPagingResponse(method) -%}
+{%- if method.pagingResponseType.includes('TYPE_STRING') -%}
```

Comment:
Can we use `typescriptType` macro here?

Reply:
`typescriptType` still returns type, but we need class to create new object. For `TYPE_BYTES`, return value from `typescriptType` returns `buffer`, but for `generateSampleMessage`,we need `new Buffer('')`. I was thinking about a more efficient way, do you have some suggestion here? Thanks!